src/pages/PokemonList.jsx

```diff
@@ -43,6 +46,10 @@ export const PokemonList = () => {
 
```

Comment:
The handling of request doesn't allow the use of pre fetched data (our pokemon easter egg names are not fetched from the pokeapi instead they are stored on the website). Is there a better way to handle prefetch data?